sdks/java/harness/src/main/java/org/apache/beam/fn/harness/control/BeamFnControlClient.java

```diff
@@ -167,9 +191,15 @@ public void sendInstructionResponse(BeamFnApi.InstructionResponse value) {
     outboundObserver.onNext(value);
   }
 
+  private void sendErrorResponse(Error e) {
+    outboundObserver.onError(e);
```

Comment:
This will close the channel from the callers side. Add todo to consider having the SDK attempt to recover itself by relying on the container to restart it.

We should also convert this to a StatusException so that gRPC translates it correctly when it is transmitted.

Reply:
Done.